project_start/express-api/server.js

```diff
@@ -32,11 +32,13 @@ const Parse = require('parse/node');
 Parse.initialize("jf8fBQCKtSE8fxxzMlARZZYxGgbMwLA2l9tAfwSU", "z25hAbCBiOVPkYzHIJt8PXLjZxKTDhsuvhMaVtuM")
 Parse.serverURL = "https://parseapi.back4app.com/"
 
+const baseUrl = process.env.NODE_ENV === "production" ? "https://whispering-castle-13172.herokuapp.com" : "http://localhost:8888";
+const baseRedirectUrl = process.env.NODE_ENV === "production" ? process.env.VERCEL_URL : "http://localhost:3000";
 
 var client_id = 'dde109facc9446bd95991893064d1a5c'; // Your client id
 var client_secret = 'bcdd6a7acf314244abb9063240a8599e'; // Your secret
-var redirect_uri = 'http://localhost:8888/callback'; // Your redirect uri
-
```

Comment:
Just a suggestion : In case any variable is used by multiple js files, you can consider creating a js file that contains all the shared constants and refer it from there

Reply:
Got it!